src/tracer.js

```diff
@@ -462,19 +464,35 @@ module.exports.filterTrace = function filterTrace(traceObject, ignoredKeys, remo
         for (let i = 0; i < ignoredKeys.length; i += 1) {
             const predicate = ignoredKeys[i];
             if (typeof predicate === 'string' &&
-            config.processIgnoredKey(predicate) === config.processIgnoredKey(key)) {
+            predicate === config.processIgnoredKey(key)) {
```

Comment:
moved the processing of the config to when the config is set (instead of each time)